iwyu_path_util.cc

```diff
@@ -149,6 +149,9 @@ string NormalizeFilePath(const string& path) {
   string result = CanonicalizeFilePath(path);
   while (StripLeft(&result, "./")) {
   }
+  // Don't use StripRight as we might have 'foo/../' here
```

Comment:
Oh, OK. You can't use `StripRight("./")` because it might cut half of a trailing `"../"` You can probably make the comment clearer here...